internal/kafka/test/helper.go

```diff
@@ -60,12 +60,18 @@ func NewKafkaHelper(t *testing.T, server *httptest.Server) (*test.Helper, *publi
 
 func NewKafkaHelperWithHooks(t *testing.T, server *httptest.Server, configurationHook interface{}) (*test.Helper, *public.APIClient, func()) {
 	h, teardown := test.NewHelperWithHooks(t, server, configurationHook, kafka.ConfigProviders(), di.ProvideValue(environments.BeforeCreateServicesHook{
-		Func: func(dataplaneClusterConfig *config.DataplaneClusterConfig, kafkaConfig *config.KafkaConfig, observabilityConfiguration *observatorium.ObservabilityConfiguration, kasFleetshardConfig *config.KasFleetshardConfig) {
+		Func: func(dataplaneClusterConfig *config.DataplaneClusterConfig, kafkaConfig *config.KafkaConfig, observabilityConfiguration *observatorium.ObservabilityConfiguration, kasFleetshardConfig *config.KasFleetshardConfig, providerConfig *config.ProviderConfig) {
 			kafkaConfig.KafkaLifespan.EnableDeletionOfExpiredKafka = true
 			observabilityConfiguration.EnableMock = true
 			dataplaneClusterConfig.DataPlaneClusterScalingType = config.NoScaling // disable scaling by default as it will be activated in specific tests
 			dataplaneClusterConfig.RawKubernetesConfig = nil                      // disable applying resources for standalone clusters
 			dataplaneClusterConfig.EnableDynamicScaleUpManagerScaleUpTrigger = false
+
+			// enable only aws for integration tests
+			awsProvider, _ := providerConfig.ProvidersConfig.SupportedProviders.GetByName("aws")
+			providerConfig.ProvidersConfig = config.ProviderConfiguration{
```

Comment:
Did we verify if this change is propagated to the tests effectively when they run??

Reply:
Yes, modification of the providers config here is passed to the tests. I've verified it by providing an empty supported providers list which causes the tests to fail with:

`Unable to initialize testing environment: expected 1 default provider in provider list, got 0`

Reply:
:+1: